### PR TITLE
Add some details to wiz docu

### PR DESCRIPTION
**Pull request by sbidy**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/44779
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/wiz.markdown [hunk 1]
```diff
@@ -16,5 +16,29 @@ ha_platforms:
...
+1. Open the WiZ app on your phone
+2. Click on the settings menu in the upper-left corner
+3. Scroll down to the security settings
+4. Enable the switch **Allow local communication**
+
+## Using discovery in container
+
+For discovering WiZ devices in your network, the Home Assistant container must be able to send and receive UDP broadcast packages.
+This requires that the container has access to the host network. This can be achieved by setting `--network host` as start parameter for the Home Assistant Docker container.
+More details can be found in the [official Docker documentation](https://docs.docker.com/network/host/).
```
Comment (frenck) [inline source/_integrations/wiz.markdown]: This section can be removed, we do not support running it without host networking.

```suggestion
```

File: source/_integrations/wiz.markdown [hunk 2]
```diff
@@ -16,5 +16,29 @@ ha_platforms:
...
+
+The integration needs to communicated locally with the WiZ device. This setting **Allow local communication** can be disabled or enabled in WiZ app.
+This setting should be enabled by default.
+
+Steps to enable:
+
+1. Open the WiZ app on your phone
+2. Click on the settings menu in the upper-left corner
+3. Scroll down to the security settings
+4. Enable the switch **Allow local communication**
```
Comment (frenck) [inline source/_integrations/wiz.markdown]: Should this be listed as prerequisites above the config flow instructions?

File: source/_integrations/wiz.markdown [hunk 3]
```diff
@@ -16,5 +16,29 @@ ha_platforms:
...
 
 The WiZ integration allows you to control your WiZ lights and smart sockets.
+The devices are set up through your Wi-Fi network and don't need any additional bridge or gateway.
 
 {% include integrations/config_flow.md %}
+
+## Connect WiZ devices to your network
+
+To connect a WiZ device to your Wi-Fi network, please follow the instructions in the [WiZ app](https://www.wizconnected.com/en/consumer/app/) (available for iOS and Android).
+If you have further questions, please have a look into the [WiZ Support Page](https://www.wizconnected.com/en/consumer/support/).
```
Comment (frenck) [inline source/_integrations/wiz.markdown]:
```suggestion
To connect a WiZ device to your Wi-Fi network, please follow the instructions in the [WiZ app](https://www.wizconnected.com/en/consumer/app/) (available for iOS and Android).
If you have further questions, please have a look at the [WiZ Support Page](https://www.wizconnected.com/en/consumer/support/).
```

File: source/_integrations/wiz.markdown [hunk 4]
```diff
@@ -16,5 +16,29 @@ ha_platforms:
...
+
+### Enable local connectivity
+
+The integration needs to communicated locally with the WiZ device. This setting **Allow local communication** can be disabled or enabled in WiZ app.
```
Comment (frenck) [inline source/_integrations/wiz.markdown]:
```suggestion
The integration needs to communicate locally with the WiZ device. This setting **Allow local communication** can be disabled or enabled in WiZ app.
```

File: source/_integrations/wiz.markdown (hunk 2 above)
Comment (frenck) [inline source/_integrations/wiz.markdown]:
```suggestion
1. Open the WiZ app on your phone.
2. Click on the settings menu in the upper-left corner.
3. Scroll down to the security settings.
4. Enable the switch **Allow local communication**.
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/wiz.markdown (hunk 2 above)
Comment (sbidy) [reply source/_integrations/wiz.markdown]: The connection part should be above. Because this step is necessary if you plug in a device the first time.

**Review by frenck — DISMISSED** [review]
Thanks, @sbidy 👍

**Review by frenck — APPROVED** [review]

**Pull request merged**